less/variables.less

```diff
@@ -269,6 +269,14 @@
 @spacing-3-tablet: 72px;
 @spacing-3-desktop: 96px;
 
+@spacing-lg-mobile: 80px;
```

Comment:
Where do these sizes come from?  There is a system to the existing sizes:

16 -> 24 -> 32
32 -> 48 -> 64
48 -> 72 -> 96

The next logical sizes would be

64 -> 96 -> 128
80 -> 120 -> 160

Hard to imaging why we'd ever want spacing of 112 on mobile?

Reply:
Could you show me the designs that motivate this?

Reply:
None of these examples use the 224px variant though, right?  Is that one necessary?  If not would prefer to stick to the system.  One thing I try to bear in mind is what will happen when requirements change, by 'skipping' some we can't bring those in later, as we'd need '4.5' or something, which won't work.

Reply:
112px of vertical on mobile would be a lot.

Reply:
Agree that jump the system couldn´t be consistent. Designers asked for the big ones as well to have more flexibility. So I propose these 4 new vertical sizes.
64 -> 96 -> 128
80 -> 120 -> 160
96 -> 144 -> 192
112 -> 168 -> 224

Reply:
Ok.  Don't seem necessary to me, but I'm fine with it :-)